snooty/rstspec.toml

```
@@ -1166,19 +1166,23 @@ type = {link = "https://mongodb.github.io/mongo-csharp-driver/2.10/apidocs/html/
 
 [role.java-docs]
 help = """Link to the Java driver's documentation."""
-type = {link = "http://mongodb.github.io/mongo-java-driver/4.1/%s"}
+type = {link = "http://mongodb.github.io/mongo-java-driver/4.2/%s"}
```

Could make this secure to standardize with the other URLs

```suggestion
type = {link = "https://mongodb.github.io/mongo-java-driver/4.2/%s"}
```